Makefile

```diff
@@ -106,11 +107,20 @@ verify-generate: generate
 
 # Build the container image
 image-build: test
```

Comment:
Do we need these at all anymore?

Reply:
With ko, don't think so. Same goes for the Dockerfile.